### DISCUSSION TITLE
[预览] v2.6.1

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#3972] [分享 / Share]
包含 v2.6.1 的所有更新内容.

- 提高了 `自定义顶栏` 搜索栏中搜索 av 号 / BV 号结果的优先级, 输入编号后直接回车就可以前往视频.
- 添加跨域存储功能 (`crossOriginLocalStorage`), 支持搜索历史跨域同步. (#3893)
- 新增功能 `弹幕转义`. (#3470)
> 将弹幕中的 `\n` 替换为真实的换行, 注意这可能导致原先不重叠的弹幕发生重叠.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.6.1-preview'>[预览] v2.6.1</a>.</em>

**Comment by Geeyun-JY3** [comment]
看到第一条还以为实现了 #677 的 2，实测后发现真的就只是调整了优先级😂

**Reply by chennanyue** [reply]
播放前显示封面 在视频暂停时仍然显示封面. (#3698)  
这一条在这个版本又有了。

**Comment by the1812** [comment]
`crossOriginLocalStorage` 还是有点 bug (
